app/models/user.rb

```diff
@@ -19,4 +19,12 @@ def self.top_spending_customers(from, to)
     customers_orders(from, to).map { |customer| customers_order_amount.store(customer, customer.payments.sum(&:total_amount)) }
     customers_order_amount.sort_by { |customer, amount| -amount }
   end
+
+  def soft_delete
```

Comment:
let's use this gem instead
https://github.com/jhawthorn/discard

Reply:
done